### ISSUE TITLE
UIX: The entire UIX canvas disappears when the Slot is moved

### ISSUE THREAD

**Issue by sweshelo** [Yellow-Dog-Man/Resonite-Issues#507]
### Describe the bug?

When you move a part of the UIX canvas, for example some elements designed as a Prefab, to a non-Active Slot, the entire UIX canvas disappears. The canvas that is no longer drawn by this operation will not be permanently restored when you duplicate or Undo it, for example.

### To Reproduce

1. Create a UIX canvas
2. Create a Non-Active Slot
3. Move any Slot in the UIX canvas to the Slot created in 2.
4. The entire UIX canvas will disappear and only the collider will remain

### Expected behavior

Only simple Slot moves should take place. Visually, only the moved Slot should disappear.

### Screenshots

![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/75022007/3e4bfc34-f780-462c-9517-9b8ebf1662c7)

### Resonite Version Number

2023.10.20.831

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Meta Quest 2

### Log Files

[SW-DESKTOP - 2023.10.20.831 - 2023-11-03 03_32_02.log](https://github.com/Yellow-Dog-Man/Resonite-Issues/files/13243353/SW-DESKTOP.-.2023.10.20.831.-.2023-11-03.03_32_02.log)

### Additional Context

I pray that a means will be provided to redraw the canvases that were unfortunately lost due to this issue.
Many parts of this bug report use machine translation by DeepL.

### Reporters

Sweshelo

**Comment by shadowpanther** [comment]
Yeah, the UIX building is a bit janky. Sometimes when duplicating slots the layout would not be what you'd expect. You'd have to save/load the item or duplicate it to restore the proper layout.

**Comment by sweshelo** [comment]
Indeed. Worse, the problem described in this Issue cannot be solved by save/load or duplicate. In other words, you can't "restore" it after it's gone, so you have to avoid this problem by, for example, making the destination Active, then moving it, then duplicating the entire Object, then changing duplicated one to Non-Active.
